api/client/client.go

```diff
@@ -3329,3 +3329,22 @@ func (c *Client) GetHeadlessAuthentication(ctx context.Context, id string) (*typ
 	}
 	return headlessAuthn, nil
 }
+
+// GetAssistantMessages retrieves a headless authentication by id.
```

Comment:
headless authentication?

Reply:
copy -> paste -> update. The last failed. Fixing. Thanks